2024-06-11 14:25:01.631 [info] (Client) Pylance async client (2024.6.1) started with python extension (2024.8.0)
2024-06-11 14:25:02.408 [info] [Info  - 2:25:02 PM] (894) Pylance language server 2024.6.1 (pyright version 1.1.364, commit 0618acc5) starting
2024-06-11 14:25:02.408 [info] [Info  - 2:25:02 PM] (894) Server root directory: file:///home/ndao/.vscode-server/extensions/ms-python.vscode-pylance-2024.6.1/dist
2024-06-11 14:25:02.427 [info] [Info  - 2:25:02 PM] (894) Starting service instance "ndao"
2024-06-11 14:25:02.471 [info] Auto-Indent enabled
2024-06-11 14:25:02.497 [info] Auto-Indent enabled
2024-06-11 14:25:02.501 [info] [Info  - 2:25:02 PM] (894) Setting pythonPath for service "ndao": "/bin/python3"
2024-06-11 14:25:02.502 [info] [Info  - 2:25:02 PM] (894) Setting environmentName for service "ndao": "3.8.10 (global)"
2024-06-11 14:25:02.549 [info] [Info  - 2:25:02 PM] (894) Assuming Python version 3.8.10.final.0
2024-06-11 14:25:02.817 [info] [Info  - 2:25:02 PM] (894) No source files found.
2024-06-11 14:25:02.824 [info] Auto-Indent enabled
2024-06-11 14:25:03.219 [info] [Info  - 2:25:03 PM] (894) Background analysis(1) root directory: file:///home/ndao/.vscode-server/extensions/ms-python.vscode-pylance-2024.6.1/dist
2024-06-11 14:25:03.223 [info] [Info  - 2:25:03 PM] (894) Background analysis(1) started
